### DISCUSSION TITLE
Store Procedure parameters in Swagger

### DISCUSSION THREAD

**Discussion by fboucher** [Azure/data-api-builder#1886] [Q&A] [answered]
I declared a Store Procedure for my MsSQL database in my config file and it works well when doing an HTTP request
```
https://localhost:5001/api/GetWeaponManoeuvres?weaponId=2
```

However, I was wondering why Swagger didn't seem to see it?
![image](https://github.com/Azure/data-api-builder/assets/2404846/54e2d080-5c08-44d9-84e9-f9863dc0fbea)

Is it normal, or I'm doing something wrong?

Here a section of my dab-config.json
```json
    "GetWeaponManoeuvres": {
      "source": {
        "object": "dbo.GetWeaponManoeuvres",
        "type": "stored-procedure",
        "parameters": {
          "weaponId":"1"
        }
      },
      "rest": {
        "enabled": true,
        "methods": [
          "get"
        ]
      },
     .....
    }
```

**Comment by severussundar** [comment, marked as answer]
Hi @FBoucher, thanks for reporting. 

[PR #1769 ](https://github.com/Azure/data-api-builder/pull/1769) adds the capability to display the parameters of a stored procedure in the Swagger UI.

This will be available from Data API builder version 0.10.* (the next upcoming version)

**Reply by fboucher** [reply]
Awesome! 
thanks
